prover/src/domain.rs

```diff
@@ -23,6 +23,12 @@ pub struct StarkDomain<B: StarkField> {
 
     /// Offset of the low-degree extension domain.
     domain_offset: B,
+
+    /// [g^i for i in (0..ce_domain_size)] where g is the constraint evaluation domain generator.
+    pub domain_g: Vec<B>,
+
+    /// A mapping from adjustment degrees to domain_offset^adjustment_degree.
+    pub adj_map: BTreeMap<u32, B>,
```

Comment:
nit: I'd probably name it `degree_adj_map` to make it a bit more descriptive.

Also, I wouldn't make this public and would instead add accessor functions.